batch.py

```diff
@@ -467,7 +480,7 @@ def summarise_association_results(
     from multipy.fdr import qvalue
 
     pv_all_df = pd.concat(
-        [pd.read_csv(AnyPath(output_path(pv_df)), index_col=0) for pv_df in pv_dfs]
+        [pd.read_csv(to_path(pv_df), index_col=0) for pv_df in pv_dfs if to_path(pv_df).exists()]
     )
```

Comment:
@annacuomo I think this is the fix you need (some of the pv_file paths in this list won't have been made, because the plink files were empty)